plugins/guests/debian/cap/configure_networks.rb

```diff
@@ -11,6 +11,16 @@ class ConfigureNetworks
 
         def self.configure_networks(machine, networks)
           machine.communicate.tap do |comm|
+            main_interface = ''
+            comm.execute(%q!ip route | awk '$1=="default"{print $NF;exit}'!) do |type, data|
+              main_interface = data.chomp if type == :stdout
+            end
+            available_interfaces = []
+            comm.execute("ls /sys/class/net | grep -v -E '^(lo$|docker|lxc)'") do |type, data|
```

Comment:
I think `^(lo$|docker|lxc)` should be `^(lo$|docker|lxc|lxd)`. The lxd project recently started calling their bridge interfaces lxdbrX instead of lxcbrX.